textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/join/JoinTestConstants.java

```diff
@@ -0,0 +1,107 @@
+package edu.uci.ics.textdb.dataflow.join;
+
+import java.util.Arrays;
+import java.util.List;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.FieldType;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.common.Schema;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.IntegerField;
+import edu.uci.ics.textdb.common.field.StringField;
+import edu.uci.ics.textdb.common.field.TextField;
+
+public class JoinTestConstants {
```

Comment:
This class defines the schema and data used in Join tests. Now most test cases have three classes: a class for test logic, a class for helper functions, and a class for test constants.